datalad_ria/tests/utils.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+
+import os
+from pathlib import Path
+import subprocess
+
+
+def assert_ssh_access(
+    host: str,
+    port: str,
+    login: str,
+    seckey: str,
+    path: str,
+    localpath: str | None = None,
+):
```

Comment:
I think something here is off:
(from appveyor)
```
=================================== ERRORS ====================================
________________________ ERROR collecting test session ________________________
C:\Python39-x64\lib\site-packages\_pytest\config\__init__.py:641: in _importconftest
    mod = import_path(conftestpath, mode=importmode, root=rootpath)
C:\Python39-x64\lib\site-packages\_pytest\pathlib.py:567: in import_path
    importlib.import_module(module_name)
C:\Python39-x64\lib\importlib\__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
<frozen importlib._bootstrap>:1030: in _gcd_import
    ???
<frozen importlib._bootstrap>:1007: in _find_and_load
    ???
<frozen importlib._bootstrap>:986: in _find_and_load_unlocked
    ???
<frozen importlib._bootstrap>:680: in _load_unlocked
    ???
C:\Python39-x64\lib\site-packages\_pytest\assertion\rewrite.py:178: in exec_module
    exec(co, module.__dict__)
C:\Python39-x64\lib\site-packages\datalad_ria\conftest.py:7: in <module>
    from .tests.fixtures import (
C:\Python39-x64\lib\site-packages\datalad_ria\tests\fixtures.py:9: in <module>
    from datalad_ria.utils import (
C:\Python39-x64\lib\site-packages\datalad_ria\utils.py:4: in <module>
    port: str | None = None,
E   TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```

Reply:
Fixed.